src/core/agent/services/identifierService.ts

```diff
@@ -93,6 +94,10 @@ class IdentifierService extends AgentService {
   async getIdentifier(
     identifier: string
   ): Promise<GetIdentifierResult | undefined> {
+    const isKeriOnline = this.signifyApi.getKeriaOnlineStatus();
```

Comment:
This is an annoying ask but is there any way we can do something more elegant here such as TypeScript decorators. So we maybe have a `@OnlineOnly` decorator or something that adds this check before the function is called.

Might be awkward but just give it a bit of a look. It feels like a lot of boilerplate to put this check like this in every call